app/models/user.rb

```diff
@@ -0,0 +1,24 @@
+class User < ActiveRecord::Base
+  validates :email, :name, :uid, :provider, presence: true
```

Comment:
Cool, that is nicer to read for sure.

On Tue, May 24, 2016 at 3:17 PM, Riley Spicer <notifications@github.com>
wrote:

> In app/models/user.rb
> <https://github.com/Ada-C5/tunes-takeout/pull/5#discussion_r64484823>:
>
> > @@ -0,0 +1,24 @@
> > +class User < ActiveRecord::Base
> > +  validates :email, :name, :uid, :provider, presence: true
>
> You may know this, but you can also use validates_presence_ofinstead of
> validates and presence: true
>
> I think the former is more readable, when you are just doing these really
> simple validations. Just showing an alternate way; your way totally works :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Ada-C5/tunes-takeout/pull/5/files/44c4d956d96ca833d6958fa91f68e369fa694b1c#r64484823>
>